### PR TITLE
Prevent numerical menu labels from being localized.

### PR DESCRIPTION
**Pull request by pineapplespatula**
fix for issue #3256

### CODE REVIEW

**Comment by DivyaPrabhakar** [comment]
- This is failing CI tests
- I checked out this branch and Menu Commands have no search results now

**Comment by pineapplespatula** [comment]
this is rebased now. I dont see any difference between master and this branch in functionality.

**Comment by volfied** [comment]
Couple weeks/months down the line, one of us is going to look at this commit message and say WTF, and will have to read the code change to see what changed. It would help everyone if the commit message was a bit more descriptive of what it accomplished:

> Prevent numerical menu labels from being localized.

File: src/js/actions/search/commands.js [hunk 1]
```diff
@@ -50,18 +50,20 @@ define(function (require, exports) {
             menuTree;
 
         parts.forEach(function (part) {
-            nlsPath = nlsPath + "." + part;
-            menuTree = nls.localize(nlsPath);
+            if (!_.isNumber(parseInt(part, 10))) {
```
Comment (volfied) [inline src/js/actions/search/commands.js]: There is no `else` statement here, do we not need to append the number itself to the resultPath at the end?

Comment (pineapplespatula) [inline src/js/actions/search/commands.js]: I'm not sure, it doesn't seem like it should matter but also it wont hurt

**Comment by iwehrman** [comment]
In whose court is the proverbial ball here? Looks like the last commit is 10 days old...

**Pull request merged**